scripts/buyer-burner/deployETH.ts

```
@@ -0,0 +1,92 @@
+import { ethers, upgrades } from 'hardhat';
+import { BuyerBurnerCCIPCaller, BuyerBurnerSwapper, SwarmBuyerBurner } from '../../typechain';
```

ESLint

[error] 'BuyerBurnerCCIPCaller' is defined but never used. Allowed unused vars must match /_/u.

(@typescript-eslint/no-unused-vars)